app/renderer/ui/main-view/menubar-view/window-menu-bar.vue

```diff
@@ -50,9 +54,28 @@ const onMinimizeClicked = () => {
   PLMainAPI.windowProcessManagementService.minimize(Process.renderer);
 };
 
+const onUnmaximizeClicked = () => {
+  PLMainAPI.windowProcessManagementService.unmaximize(Process.renderer);
+};
+
 const onMaximizeClicked = () => {
   PLMainAPI.windowProcessManagementService.maximize(Process.renderer);
 };
+
+
+disposable(
```

Comment:
Because I found that users can maximize or unmaximize the window by the blank header. Double-click or drag that.
So listening to the event can handle all conditions.

![image](https://github.com/Future-Scholars/paperlib/assets/27353191/1a26d04b-fccf-4dc7-a397-1cbe0537397f)

Reply:
make sence.